lib/Twig/Extension/Core.php

```diff
@@ -1463,15 +1463,25 @@ function twig_get_attribute(Twig_Environment $env, Twig_Source $source, $object,
         throw new Twig_Error_Runtime('Accessing Twig_Template attributes is forbidden.');
     }
 
+    static $extensionSandbox = array();
```

Comment:
would it better introduce private non-static class property `hasSandboxExtension`?